src/controllers/auth.controller.js

```diff
@@ -0,0 +1,107 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { User } from '../modules/User.js';
+import { userService } from '../service/user.service.js';
+import bcrypt from 'bcrypt';
+import { Console } from '../utils/Console.js';
```

Comment:
The import path for `Console.js` is incorrect. The directory `utils` should be used instead of `untils`.